modules/async-icq/module.go

```diff
@@ -142,13 +160,13 @@ func (am AppModule) ExportGenesis(ctx sdk.Context, cdc codec.JSONCodec) json.Raw
 }
 
 // ConsensusVersion implements AppModule/ConsensusVersion.
-func (AppModule) ConsensusVersion() uint64 { return 1 }
+func (AppModule) ConsensusVersion() uint64 { return 2 }
 
 // BeginBlock implements the AppModule interface
-func (am AppModule) BeginBlock(_ sdk.Context, _ abci.RequestBeginBlock) {
+func (am AppModule) BeginBlock(sdk.Context, abci.RequestBeginBlock) {
```

Comment:
out of curiosity, what is the difference between just defining the type for the parameters like this and omitting the underscore vs. using the underscore? 

i know typically you use the underscore to signify that you are satisfying some interface implementation but that the parameter is not actually being used.

Reply:
You should use `_` more as a placeholder if you may use it in the future, just not now. Since the `BeginBlock`er is not ever going to be used for this module, just the types satisfy the interface

Reply:
til, ty!